connect/runtime/src/test/java/org/apache/kafka/connect/integration/OffsetsApiIntegrationTest.java

```diff
@@ -927,6 +955,53 @@ private Map<String, String> baseSourceConnectorConfigs() {
         return props;
     }
 
+    /**
+     * Wait for a consumer group to exist and have the expected number of members.
+     * @param kafkaCluster the Kafka cluster that should host the group; may not be null
+     * @param groupId the ID of the group; may not be null
+     * @param expectedMembers the exact number of members that should be in the group; may not be null
+     * @throws InterruptedException if the thread is interrupted while waiting for the consumer
+     * group to form
+     */
+    private void verifyConsumerGroupMembers(
```

Comment:
TODO: If this method is useful, it will obviate the existing `modifySinkConnectorOffsetsWithRetry` method, which we will then be able to remove.